swap/htlc.go

```diff
@@ -311,10 +315,10 @@ func (h *Htlc) AddTimeoutToEstimator(estimator *input.TxWeightEstimator) error {
 			trHtlc.InternalPubKey, timeoutLeaf, successLeafHash[:],
 		)
 
-		estimator.AddTapscriptInput(maxTimeoutWitnessSize, tapscript)
+		estimator.AddTapscriptInput(lntypes.WeightUnit(maxTimeoutWitnessSize), tapscript)
```

Comment:
Let's change `MaxTimeoutWitnessSize()` and `MaxSuccessWitnessSize` to return `lntypes.WeightUnit` and we can avoid conversions here. Also it would be clear from the interface what the units of the results are.